src/orion/benchmark/__init__.py

```diff
@@ -120,11 +120,11 @@ def status(self, silent=True):
         return benchmark_status
 
     def analysis(self):
-        """Return all the assessment figures"""
-        figures = []
+        """Return all the assessment figures with format as {assessment_name: {figure_name: figure_object}}"""
+        figures = dict()
         for study in self.studies:
             figure = study.analysis()
-            figures.append(figure)
+            figures[study.assess_name + "_" + study.task_name] = figure
```

Comment:
I think using a nested dict structure would make it easier to navigate through the figures.